FirebaseVertexAI/Sources/GenerateContentRequest.swift

```diff
@@ -15,7 +15,7 @@
 import Foundation
 
 @available(iOS 15.0, macOS 12.0, macCatalyst 15.0, tvOS 15.0, watchOS 8.0, *)
-struct GenerateContentRequest {
+struct GenerateContentRequest: Sendable {
```

Comment:
nit: I wonder if we could make `GenerativeAIRequest` conform to `Sendable` (haven't tried so it may introduce a lot more warnings).

Reply:
Done, with a few related changes.